data/goal_src/jak1/pc/progress-pc.gc

```diff
@@ -102,6 +108,7 @@
 
 (define *main-options-pc* (new 'static 'boxed-array :type game-option
     (new 'static 'game-option :option-type (game-option-type menu) :name (game-text-id game-options) :scale #t :param3 (game-option-menu game-settings))
+    (new 'static 'game-option :option-type (game-option-type menu) :name (game-text-id mod-options) :scale #t :param3 (game-option-menu mod-options))
```

Comment:
Moved Mod Options up a couple levels, as a sibling to Game/Graphics/Sound Options (tested on title and in-game menus)